src/js/view/controls/controls.js

```diff
@@ -117,7 +117,7 @@ export default class Controls extends Events {
         if (!this.displayContainer) {
             const displayContainer = new DisplayContainer(model, api);
 
-            displayContainer.buttons.display.on('click', () => {
+            displayContainer.buttons.display.on('click enter', () => {
```

Comment:
Can we use `addClickAction` here?

Also UI instances should always be returned and stored as a property of their parent instance (Controls). This is so that we destroy the UI instance when destroying the parent to remove all listeners.

Reply:
Same as above. These are destroyed when `displayContainer.destroy` is called:

<img width="415" alt="image" src="https://user-images.githubusercontent.com/2903955/121192942-06543000-c833-11eb-9084-68c219d875fb.png">